middle_end/flambda/types/env/typing_env_extension.rec.ml

```diff
@@ -40,6 +40,9 @@ let print ppf t =
     "@[<hov 1>(equations@ @[<v 1>%a@])@]"
     print_equations t.equations
 
+let print_with_cache ~cache:_ ppf t =
```

Comment:
Is this function used anywhere ?

Reply:
Sorry, I guess it's part of the `Expr_std.S` signature.

Reply:
I remove Exp_std. And removed this.